services/sidecar/src/simcore_service_sidecar/log_parser.py

```diff
@@ -101,6 +101,7 @@ async def _monitor_docker_container(
                 log_type, parsed_line = await parse_line(line)
                 await log_cb(log_type, parsed_line)
                 await writer(f"{log_type.name}: {parsed_line}")
+                await asyncio.sleep(1)
```

Comment:
MINOR: I would create doc const here ... it is far more readable
```python
# top 
UNTIL_GET_LOG_WRITTEN = 1

# ...
await writer( ...) 
await asyncio.sleep( UNTIL_GET_LOG_WRITTEN )
```

Reply:
no actually that sleep is useless. removed.